http-client/src/test/java/io/airlift/http/client/TestHttpClientBinder.java

```diff
@@ -236,8 +236,8 @@ public void configure(Binder binder)
         assertSame(injector.getInstance(Key.get(HttpClient.class, FooAlias2.class)), client);
         assertSame(injector.getInstance(Key.get(HttpClient.class, FooAlias3.class)), client);
 
-        // a private pool should not be registered for these clients
-        assertNull(injector.getExistingBinding(Key.get(JettyIoPoolManager.class, FooClient.class)));
+        // a private pool should be registered for the FooClient only
```

Comment:
Let's remove the word "private" since that concept no longer exists. Also change the comment to
```java
// a pool should be registered for the FooClient only (not for the aliases)

Reply:
Similarly, the test method names below probably need to be updated now